src/core/server/graph/schema/schema.graphql

```diff
@@ -2184,6 +2184,17 @@ enum DIGEST_FREQUENCY {
   HOURLY
 }
 
+"""
+EmbedPreferences are the user's preferences around embed stream behaviour.
+"""
+type EmbedPreferences {
```

Comment:
We've renamed the "embed" to "media", can you rename instances of "embed" to "media"?